src/debug.c

```diff
@@ -2542,41 +2592,48 @@ static int is_thread_ready_to_signal(pid_t pid, pid_t tid, int sig_num) {
  * Writes into tids the tids of these threads.
  * If it fails, returns 0.
 */
-static size_t get_ready_to_signal_threads_tids(pid_t pid, int sig_num, pid_t tids[TIDS_MAX_SIZE]) {
-    /* Initialize the path the process threads' directory. */
-    char path_buff[MAX_BUFF_LENGTH];
-    snprintf(path_buff, MAX_BUFF_LENGTH, "/proc/%d/task", pid);
+static size_t get_ready_to_signal_threads_tids(int sig_num, pid_t tids[TIDS_MAX_SIZE]) {
+    /* Open /proc/<pid>/task file. */
+    char path_buff[PATH_MAX] = "/proc/";
```

Comment:
```suggestion
    char path_buff[PATH_MAX];
```
Leftover? Seems not necessary anymore as it will overridden on the next line